pkg/client/cli/cmd_preview.go

```diff
@@ -16,6 +16,8 @@ import (
 // fields in the given 'spec'.  If 'prefix' is given, long-flag names are prefixed with it.
 func addPreviewFlags(prefix string, flags *pflag.FlagSet, spec *manager.PreviewSpec) {
 	flags.BoolVarP(&spec.DisplayBanner, prefix+"banner", "b", true, "Display banner on preview page")
+	flags.StringToStringVarP(&spec.InjectHeaders, prefix+"add-headers", "", map[string]string{},
```

Comment:
Why did you call it "add" externally, but "inject" internally? Nit: IMO, whichever word you think is clearer should be the word used in both places.

Reply:
Change request: Taking a lesson from Emissary history, it's probably worth clarifying this as "add-request-headers"; so that no one (confused Ambassador Labs engineer or confused user alike) mistakes it as adding response headers.

Reply:
Yeah, 💯. I was trying different names: `inject`, `add`, `add-request`, etc. and settled on `add` simply to make the CLI arg shorter and not skew the `--help` output. I do agree `--preview-url-add-request-headers` makes the most sense.